internal/pkg/util/util.go

```diff
@@ -315,3 +315,45 @@ func CopyUIDGID(source string, dest string) error {
 	err = os.Chown(dest, UID, GID)
 	return err
 }
+
+func SplitEscaped(input, delim, escape string) []string {
+	var ret []string
+	str := ""
+	for i := 1; i < len(input); i++ {
+		str += string(input[i-1])
+		if string(input[i]) == delim && string(input[i-1]) != escape {
+			i++
+			ret = append(ret, str)
+			str = ""
+		}
+		if string(input[i]) == escape {
+			i++
+		}
+
+	}
+	str += string(input[len(input)-1])
+	ret = append(ret, str)
+
+	return (ret)
+}
+
+func SplitPaths(input, delim string) []string {
+	var ret []string
+	str := ""
+	for i := 1; i < len(input); i++ {
+		str += string(input[i-1])
+		if (string(input[i]) == delim && string(input[i-1]) != "\\") && (IsDir(str) || IsFile(str)) {
```

Comment:
This is doing a little bit more than just string manipulation. It is validating the path.

Maybe rename the function "SplitValidPaths"?  Just a thought.

Reply:
Done. Thanks!